src/components/List/SimpleList/SimpleList.story.jsx

```diff
@@ -0,0 +1,320 @@
+import React from 'react';
+import { Add16, Close16, Edit16 } from '@carbon/icons-react';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { Button, OverflowMenu, OverflowMenuItem } from 'carbon-components-react';
+import { withReadme } from 'storybook-readme';
+
+import SimpleList from './SimpleList';
+import SimpleListREADME from './README.md';
+
+const getListItems = num =>
+  Array(num)
+    .fill(0)
+    .map((i, idx) => ({
+      id: idx + 1,
+      content: { value: `Item ${idx + 1}` },
+      isSelectable: true,
+    }));
+
+const listItemsWithEmptyRow = getListItems(5).concat({ id: 6, content: { value: '' } });
+
+const rowActions = [
+  <Edit16 onClick={action('edit')} />,
+  <Add16 onClick={action('add')} />,
+  <Close16 onClick={action('close')} />,
+];
+
+const rowActionsOverFlowMenu = [
+  <OverflowMenu flipped>
+    <OverflowMenuItem itemText="Edit" />
+    <OverflowMenuItem itemText="Add" />
+    <OverflowMenuItem itemText="Delete" />
+    <OverflowMenuItem itemText="Danger option" hasDivider isDelete />
```

Comment:
Translation?

Reply:
That's all story stuff -- the user can provide whatever text they'd like, there (we'd assume it's translated)

Reply:
sorry. True that.